examples/tensorflow/common/dataset_builder.py

```diff
@@ -119,13 +120,14 @@ def _load_tfds(self):
         return dataset
 
     def _load_tfrecords(self):
-        logger.info("Using TFRecords to load {} data.".format(self._split))
+        logger.info(f"Using TFRecords to load {self._split} data.")
 
         dataset_key = self._dataset_name.replace("/", "")
         if dataset_key in self._tfrecord_datasets:
             self._dataset_loader = self._tfrecord_datasets[dataset_key](config=self._config, is_train=self._is_train)
         else:
-            raise nncf.UnknownDatasetError("Unknown dataset name: {}".format(self._dataset_name))
+            msg = f"Unknown dataset name: {self._dataset_name}"
+            raise nncf.UnknownDatasetError(msg)
```

Comment:
We have a great pack of the nncf-specific (like `nncf.UnknownDatasetError`) errors here - nncf/errors.py. Why can't we update the wrappers there instead of changing messages everywhere?

Reply:
Which wrapper are you referring to? 
`nncf.UnknownDatasetError` is just subclass of Exception.

It's not clear for me how it possible to update error in nncf.error to follow EM rules.
Please provide example of your suggestion.